src/StateSmithTest/Output/UserConfig/TomlConfigTest_335.cs

```diff
@@ -142,7 +142,7 @@ public void TestReflectionMapping()
         var reader = new TomlReader(renderConfigAllVars, smRunnerSettings);
 
         TestHelper.ExpectPropertyCount<IRenderConfig>(8, because: "test need updating");
-        TestHelper.ExpectPropertyCount<IRenderConfigC>(7, because: "test need updating");
+        TestHelper.ExpectPropertyCount<IRenderConfigC>(8, because: "test need updating");
```

Comment:
This is the new style of specifying settings. We need to add test coverage here.

Add text like
```
UseStdBool = false
```
here: https://github.com/StateSmith/StateSmith/blob/33d5738442bab88373aa2a483bb36850de754b59/src/StateSmithTest/Output/UserConfig/TomlConfigTest_335.cs#L173

Reply:
Changed the test accordly to your suggestion

Reply:
looks good!